### ISSUE TITLE
uk-search-field text alignment and extending

### ISSUE THREAD

**Issue by ghost** [neicv/pagekit-search#15]
Is it possible to align the text in the search box from right to left aligned in any way or is it a bug I've found? HTML class does not work for me. I've tried something like **_uk-search-field_ text-align="left"** etc. I've also tried to edit the CSS file to extend the search input-field but even this way doens't work for me. 

What am I doing wrong? Is it maybe a problem of the Avion theme?

**Comment by neicv** [comment]
Hi! Did I understand you correctly what you mean "Search Widget "?

"..CSS file to extend the search.." - that "know bug" - testing func.

**Comment by mrtimsel** [comment]
Yes I think so but I am new to Pagekit. I also know about the configuration from the Pagekit manual. 

Thanks for your reply.

**Comment by neicv** [comment]
1.
In default, Pagekit use uikit 2.0  framework and search extension use too.
In choised Theme you can customize all elements of uikit. (handmake or use customizer https://getuikit.com/v2/docs/customizer.html)

css class uk-search and uk-search-field are located in:
For theme Avion: 
your-domain\packages\pagekit\theme-avion\less\uikit\search.less

2. I add custom class to get rid of uikit, but this is not completed yet...

Last relese search ext:
https://pagekit.friendly-it.ru/download

**Comment by neicv** [comment]
And for all questions about Pagekit, welcome to join 
Pagekit Forum
https://pagekit-forum.org/

**Comment by neicv** [comment]
If you want, I can add additional appearance settings in widget

**Comment by mrtimsel** [comment]
Thanks for that!
It would be great if you could add some appearance settings in widget.

**Comment by neicv** [comment]
exactly what settings do you need?

**Comment by mrtimsel** [comment]
I would need a custom font size and a way to expand the length of the search box (you know it only expands a few characters if you click into the search box in every theme I tried so far). Also, a custom placeholder would be fine but I could handle that with a text widget and HTML search capabilities. In most cases I tried the atteributes from the Pagekit manual, text widget or HTML class field but it does not work for me.
